src/Common/Common.Api/Gateways/Entities/GatewayClusterDatasource.cs

```diff
@@ -0,0 +1,39 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Microsoft.PowerBI.Common.Api.Gateways.Entities
+{
+    [DataContract]
+    public class GatewayClusterDatasource
```

Comment:
sealed class

Reply:
I would like to keep this the same as the backend code.